pkg/config/default.go

```diff
@@ -480,11 +479,11 @@ func defaultEngineConfig() (*EngineConfig, error) {
 }
 
 func defaultTmpDir() (string, error) {
-	if !useUserConfigLocations() {
+	if unshare.GetRootlessUID() == 0 {
```

Comment:
This is not the negation of `> 0`, the way `defaultConfig` vs. `defaultTmpDir` used to decide. Is that intentional?

Reply:
no, it is a mistake, I'll fix it.  Thanks